academic_research.md

```diff
@@ -0,0 +1,33 @@
+# Chapter on outcome from DPKI academic research
+
+
+### Introduction:
+As time goes by, the use of distributed PKI takes more and more importance in regard to the one of centralized PKI. This change can be (partially/mainly) attributed to the increasing importance of IT security. On the one hand, centralization, on the other, the exact opposite. Thanks to decentralization there would be the possibility of eliminating, or reducing, the impact of the single point of failure, the CA. Which is not impenetrable to cyberattacks as the 2011 [DigiNotar attack](https://www.theguardian.com/technology/2011/sep/05/diginotar-certificate-hack-cyberwar) demonstrates.
+The CA can be seen as a point of failure because in the worst scenario, it can be compromised and a malicious actor could create a fake certificate and trick the user into believing that it is a real one.
+
+------------
+
+Decentralized PKI can be divided into two main categories:
+- **Log-based DPKI**: This solution is mainly used to monitor the CA. Indeed certificates are not valid until they are logged on append-only public logs. Anyone can read the logs and so monitor the CA for suspect certificates. The first log-based solution is [Certificate Transparency](https://datatracker.ietf.org/doc/html/rfc6962.html), proposed by Google.
+- **Blockhain-based DPKI**:This solution can be split into two parts:
+    - Hierarchical: it is still a partially dependent version on the CA. It can be considered as a log-based DPKI, the only difference is the use of the blockchain. For this reason this solution is called "relaxed centralization"
+    - Web of Trust: typically replace CA with miners in public blockchains, certificates are generated by mining.
+
+In the text below we will use the term **consensus** that should be defined as follows: There are various types of consensus some more efficient than others, such as:
+- *Proof of Work* (PoW): it is used in the famous Bitcoin blockchain and many other forks of Bitcoin itself. The system requires miners to compete with each other to be the first to solve "mathematical problems". The more a miner has computing power, the more likely it is to be chosen.
+- *Proof of State* (PoS): it will be used in the near future by Ethereum. PoS eliminates the problem of having a large calculation capacity in favor of a random choice of the miner.
+- [*Byzantin Fault Tolerance*](https://ieeexplore.ieee.org/document/8433150) (BFT): there are various version of it, some more efficient than others, like [HotStuff](https://dl.acm.org/doi/pdf/10.1145/3293611.3331591). The procedure can be seen in three phases: 
```

Comment:
you are repeating the same sentence as you used before (some more efficient than others). When writing a paper it's good to use synonyms, of course the most important is the accuracy of the terms, so use a synonym if you are sure it means the same in this paticular situation.
Concerning the efficiency of consensi, you shouldn't mention it if you don't describe what you mean by that. As a general rule (that I learnt during writing master thesis in science, but I guess it applies to programming), you should not mention things you don't want to talk about, it will put you in awkward situations if your supervisors/teachers ask about it